src/spellbot/reactions.py

```diff
@@ -0,0 +1,31 @@
+import logging
+
+import discord
+
+from spellbot.assets import s
+
+logger = logging.getLogger(__name__)
+
+
+async def safe_remove_reaction(
+    message: discord.Message, emoji: str, user: discord.User
+) -> None:  # pragma: no cover
```

Comment:
We probably want to remove `pragma: no cover` here since we now have tests that cover this code 🎉